lib/dependencies/RequireContextPlugin.js

```diff
@@ -55,7 +57,7 @@ class RequireContextPlugin {
 				}).reduce((a, b) => a.concat(b), []));
 			});
 
-			contextModuleFactory.plugin("alternatives", (items, callback) => {
+			contextModuleFactory.hooks.alternatives.tapAsync("RequireContextPlugin", (items, callback) => {
```

Comment:
Use `tap` instead of `tapAsync` and remove the `callback`. Return the result instead.